src/10-webpage-publishing-with-ipfs/index.md

```diff
@@ -0,0 +1,311 @@
+--- maybe as a foot note? I don't know. It doesn't fit, even though we love Neocities. A footnore at the end of the doc should work. 
+
+The alternative -- self-hosting your content on infrastructure you control -- is even worse. Only the zealous few manage their own hardware today. And those who compromise but still dare host their own websites on virtual private servers run into a myriad of complex and brittle processes. Even very experienced web developers struggle with the complexities of setting up webservers, replicating across regions, adding load-balancers, securing machines against hackers, responding to sudden problems, and just maintaining everything working long-term. All of this requires large amounts of domain specific knowledge that newcommers find very difficult to use.
+
+"I just wanted to publish my personal website!" It is not a mystery why  most people use Facebook, Twitter, Medium, and Tumblr to publish and host their public personas, their blogs, and all their thoughts.
+
+Suggestion: This is where IPFS, the permanent web, comes in. We've been working on a solution to these issues, imagining a word where it is easier to publish web content; where it easier to host and maintain it; where links don't break or change; where old versions didn't disappear in moment; and where all of this is as simple as tweeting. All at once. 
+
+> - What if it was way easier to publish web content?
+> - What if it was way easier to host and maintain it?
+> - What if by design, links didn't break?
+> - What if by design, links didn't change when moving hosts?
+> - What if by design, old versions were all still accessible?
+> - What if managing all this was as simple as tweeting?
+
+(NOTE: Again, I think these questions come off as a bit weak. People skim through repetitive questions, only reading the right edge and not getting the rhetorical effect that you'd get in speech. Rephrase, like above.)
+
+We don't have all the solutions working yet, but we're working hard to get there. Comparatively, the IPFS Project is very young and we're still doing out all the groundwork and creating the new plumbing infrastructure. However, we can already make a big dent into these annoying problems, at least for developers and users familiar with:
```

Comment:
First time we mention the IPFS project. Obviously, this is on our website, but just saying. Link, maybe.